src/Bridges/Constraint/set_map.jl

```diff
@@ -102,7 +102,17 @@ function MOI.get(
     bridge::MultiSetMapBridge{T,S1,G},
 ) where {T,S1,G}
     mapped_func = MOI.get(model, attr, bridge.constraint)
-    func = MOI.Bridges.inverse_map_function(typeof(bridge), mapped_func)
+    func = try
+        MOI.Bridges.inverse_map_function(bridge, mapped_func)
+    catch err
+        # MapNotInvertible is thrown if the bridge does not support inverting
+        # the function. The user doesn't need to know this, only that they
+        # cannot get the attribute.
+        if err isa MOI.Bridges.MapNotInvertible
+            throw(MOI.GetAttributeNotAllowed(attr))
```

Comment:
How do we test this?

Reply:
We'll have to create a custom bridge for this in the tests I think